extra-web/nginx/autobuild/postinst

```diff
@@ -1,2 +1,27 @@
+#!/bin/bash
+
+# Set up permission for logs
 chmod 750 /var/log/nginx
 chown http:log /var/log/nginx
```

Comment:
Ditto all my notes from httpd postinst above. Don't do out-of-package operations - they don't seem necessary here.